planetiler-core/src/test/java/com/onthegomap/planetiler/PlanetilerTests.java

```diff
@@ -1684,6 +1686,46 @@ public void processFeature(SourceFeature source, FeatureCollector features) {
     }
   }
 
+  @Test
+  void testPlanetilerRunnerShapefile() throws Exception {
+    Path mbtiles = tempDir.resolve("output.mbtiles");
+    Path resourceDir = TestUtils.pathToResource("");
+
+    Planetiler.create(Arguments.fromArgs("--tmpdir=" + tempDir.resolve("data")))
+      .setProfile(new Profile.NullProfile() {
+        @Override
+        public void processFeature(SourceFeature source, FeatureCollector features) {
+          features.point("stations")
+            .setZoomRange(0, 14)
+            .setAttr("source", source.getSource());
+        }
+      })
+      .addShapefileDirectorySource("shapefile-dir", resourceDir, "shape*.zip")
+      .addShapefileSource("shapefile", resourceDir.resolve("shapefile.zip"))
+      .setOutput("mbtiles", mbtiles)
+      .run();
+
+    try (Mbtiles db = Mbtiles.newReadOnlyDatabase(mbtiles)) {
+      long fileCount = 0;
+      long dirCount = 0;
+      var tileMap = TestUtils.getTileMap(db);
+      for (var tile : tileMap.values()) {
+        for (var feature : tile) {
+          feature.geometry().validate();
+
+          switch ((String) feature.attrs().get("source")) {
+            case "shapefile" -> fileCount++;
+            case "shapefile-dir" -> dirCount++;
+          }
+        }
+      }
+
+      // Input file was copied twice into test directory, directory source should have
+      // 2x the number of features.
+      assertEquals(2 * fileCount, dirCount);
```

Comment:
might want to also assert that fileCount/dirCount are >0 just to make sure we're not saying things are ok if for some reason it doesn't read anything